doc/contrib/api/review.md

```diff
@@ -0,0 +1,15 @@
+On potential changes (including extensions) of the API/ABI please check following list:
+
+- [ ] [use case](/doc/usecases) for API exists
+- [ ] API design [review](reviews/) template is created by author of API change
+- [ ] **Before** starting to implement the change: [decisions](/doc/decisions) for API is in step `decided`
+- [ ] **After** implementing the change: [decisions](/doc/decisions) for API is in step `implemented`
+- [ ] API design is [reviewed](reviews/) by someone else, too
```

Comment:
Shouldn't the API be reviewed, **before** the decision is completed (i.e. marked as "Implemented")?